torchtext/datasets/amazonreviewpolarity.py

```diff
@@ -25,20 +27,24 @@
     'test': f'{os.sep}'.join(['amazon_review_polarity_csv', 'test.csv']),
 }
 
-_EXTRACTED_FILES_MD5 = {
-    'train': "520937107c39a2d1d1f66cd410e9ed9e",
-    'test': "f4c8bded2ecbde5f996b675db6228f16"
-}
 
 DATASET_NAME = "AmazonReviewPolarity"
 
 
 @_add_docstring_header(num_lines=NUM_LINES, num_classes=2)
 @_create_dataset_directory(dataset_name=DATASET_NAME)
-@_wrap_split_argument(('train', 'test'))
+@_wrap_split_argument(("train", "test"))
 def AmazonReviewPolarity(root, split):
-    path = _download_extract_validate(root, URL, MD5, os.path.join(root, _PATH), os.path.join(root, _EXTRACTED_FILES[split]),
-                                      _EXTRACTED_FILES_MD5[split], hash_type="md5")
-    logging.info('Creating {} data'.format(split))
-    return _RawTextIterableDataset(DATASET_NAME, NUM_LINES[split],
-                                   _create_data_from_csv(path))
+    # TODO Remove this after removing conditional dependency
+    if not is_module_available("torchdata"):
+        raise ModuleNotFoundError("Package `torchdata` not found. Please install following instructions at `https://github.com/pytorch/data`")
```

Comment:
I think once there's a stable release of torchdata published that this check will disappear. Is it still worth the test since it'll be removed soon anyway?

Reply:
I agree, this is temporary. We can probably skip adding test.

Reply:
Sounds reasonable to me.